### PR TITLE
chore(apiKeys): add list and delete live api keys

### PR DESCRIPTION
**Pull request by raul-facturapi**

### CODE REVIEW

**Review by edgtz — APPROVED** [review]

**Review by javorosas — APPROVED** [review]

File: website/openapi_v2.en.yaml [hunk 1]
```diff
@@ -4854,6 +4854,77 @@ paths:
         "500":
           $ref: "#/components/responses/UnexpectedError"
   /organizations/{organization_id}/apikeys/live:
+    get:
+      operationId: listLiveApiKey
```
Comment (javorosas) [inline website/openapi_v2.en.yaml:4858]: listLiveApiKeys

File: website/openapi_v2.en.yaml [hunk 2]
```diff
@@ -4854,6 +4854,77 @@ paths:
...
+      tags:
+        - organization
+      summary: List Live API Key
```
Comment (javorosas) [inline website/openapi_v2.en.yaml:4861]: Keys

File: website/openapi_v2.yaml [hunk 3]
```diff
@@ -4814,6 +4814,76 @@ paths:
         "500":
           $ref: "#/components/responses/UnexpectedError"
   /organizations/{organization_id}/apikeys/live:
+    get:
+      operationId: listLiveApiKey
```
Comment (javorosas) [inline website/openapi_v2.yaml:4818]: Keys

File: website/openapi_v2.yaml [hunk 4]
```diff
@@ -4814,6 +4814,76 @@ paths:
...
+      tags:
+        - organization
+      summary: Listar Live API Key
```
Comment (javorosas) [inline website/openapi_v2.yaml:4821]: Keys

**Pull request merged**